examples/oft-hyperliquid/deploy/MyHyperliquidComposer.ts

```diff
@@ -0,0 +1,98 @@
+import assert from 'assert'
+
+import { Wallet } from 'ethers'
+import { type DeployFunction } from 'hardhat-deploy/types'
+
+import { getNativeSpot, useBigBlock, useSmallBlock } from '@layerzerolabs/oft-hyperliquid-evm'
+
+const contractName_oft = 'MyHyperLiquidOFT'
+const contractName_composer = 'MyHyperLiquidComposer'
+
+const evmDecimals = 0
```

Comment:
From a devex perspective, leaving a quick comment here saying they need to fill these out instead of having to run deploy and fail might be a better experience.